pkgs/tools/typesetting/sile/default.nix

```diff
@@ -65,13 +65,12 @@ stdenv.mkDerivation rec {
     harfbuzz
     icu
     fontconfig
+    libarchive
     libiconv
+    poppler_utils
```

Comment:
It makes sense to me to at least put `libarchive` in `nativeBuildInputs`, if `poppler_utils` is mandatory for building, since the name implies these are build utilities, I'd also put them in `nativeBuildInputs`.

Reply:
Also, `libarchive` should be present in the file's function arguments at the beginning - that's what makes ofborg's evaluation fail.

Reply:
The `poppler_utils` package was previously only noted for use for checks and hence part of `checkInputs`, but it is also used while building the manual. I'm actually surprised that wasn't blocking the build in the first place, bu I suppose `checkInputs` were being supplied up front so the mistake was not caught. If (per other comments) we decide to not build the manual that can be moved back to `checkInputs`.

Reply:
> The `poppler_utils` package was previously only noted for use for checks and hence part of `checkInputs`, but it is also used while building the manual.

That's typical for `nativeBuildInputs`, and not `buildInputs`.

Is it possible to still enable checks, and disable the build of the manual, and thus also not require `libarchive`?

Reply:
That depends on what you mean by "checks". The Nix build has not been running the full test suite (`make test` runs `make busted` and `make regressions`). It takes a lot of time and introduces a TON of font and other dependencies (perl packages, etc.). All the Nix build is running is a `make check` which runs a basic end to end self check to make sure we're generating PDFs. Over and above normal run time dependencies it onl uses `pdfinfo` from `poppler_utils` to verify the resulting PDF, so that's needed for the selfcheck.

The selfcheck has no font or other crazy dependencies and is all distros should probably be running. As such `bsdtar` (from `libarchine`) is not needed. That is only used for extracting font resources used for the manual and test suite.

To disable the building of the manual, use `--without-manual`. That will still allow the self check to run without needing any of the font resource tooling.

Reply:
> The selfcheck has no font or other crazy dependencies and is all distros should probably be running.

I agree.